modules/QnA/resources/qna.js

```diff
@@ -0,0 +1,356 @@
+			var buttonWidth = Math.round(buttonHeight / 2);
+
+			var borderRadius = buttonWidth + "px 0 0 " + buttonWidth + "px";
+
+			var topOffset = (videoHeight-buttonHeight)/2 + "px";
+
+
+			var textIndent = (buttonWidth - parseInt(onVideoTogglePluginButton.css('font-size'))) / 2;
+			onVideoTogglePluginButton.css(
+                {   '-moz-border-radius': borderRadius,
+                    '-webkit-border-radius': borderRadius,
+                    'border-radius': borderRadius,
+                    height: buttonHeight + "px",
+                    width: buttonWidth + "px",
+                    top: topOffset,
+                    'line-height': buttonHeight + "px",
+                    'text-indent': textIndent + "px"});
+		},
+
+		updateQnaListHolderSize : function(){
+			var _this = this;
+			var newHeight = this.getPlayer().getInterface().height();
+			if (!_this.announcementOnlyStatus()){
+				newHeight -= _this.getQnaContainer().find('.qnaQuestionArea').height();
+			}
+			_this.getQnaContainer().find('.listHolder').height(newHeight);
+		},
+
+		bindButtons : function(){
+			var _this = this;
+			var sendButton = _this.getQnaContainer().find('.qnaSendButton');
+			sendButton.text(gM('qna-send-button-text'));
+			sendButton
+				.off('click')
+				.on('click', function(){
+					var question = _this.getQnaContainer().find('.qnaQuestionTextArea').val();
+
+					if (_this.getPlayer().isOffline() && !_this.getConfig( 'allowNewQuestionWhenNotLive' )){
+						alert(gM('qna-cant-ask-while-not-live'));
+					} else {
+					if (question !== gM('qna-default-question-box-text')) {
+						_this.KQnaService.submitQuestion(question);
+						_this.resetTextArea(textArea);
+					}
+					}
+				});
+			var cancelButton = _this.getQnaContainer().find('.qnaCancelButton');
+			cancelButton.text(gM('qna-cancel-button-text'));
+			cancelButton
+				.off('click')
+				.on('click', function(){
+					_this.resetTextArea(textArea);
+				});
+
+			var textArea = _this.getQnaContainer().find('.qnaQuestionTextArea');
+			_this.resetTextArea(textArea);
+			textArea
+				.off('focus')
+				.on('focus', function(){
+					if (textArea.val() === gM('qna-default-question-box-text')) {
+						textArea.css({'font-weight': 300});
+						textArea.val('');
+						textArea.css({'color': '#ffffff'});
+					}
+				});
+
+			textArea
+				.off('blur')
+				.on('blur', function(){
+					if (textArea.val() === '') {
+						_this.resetTextArea(textArea);
+					}
+				});
+
+			textArea.bind("mousewheel",function(ev) {
+				ev.preventDefault();
+
+				var ratio = $(this).height() / $(window).height();
+				var deltaY = ev.originalEvent.deltaY * ratio;
+				$(this).scrollTop($(this).scrollTop() - Math.round(deltaY));
+			});
+
+			textArea.bind("touchstart",function(ev) {
+				var elem = ev.target;
+				$(elem).css({'overflow':'auto'});
+			});
+		},
+
+		resetTextArea : function(textArea){
+			textArea.val(gM('qna-default-question-box-text'));
+			textArea.css({'font-weight': 300});
```

Comment:
general note - don't use direct style on element - try to use css class (addClass/removeClass jquery)